synapse/config/redis.py

```diff
@@ -33,6 +33,7 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
 
         self.redis_host = redis_config.get("host", "localhost")
         self.redis_port = redis_config.get("port", 6379)
+        self.redis_path = redis_config.get("path", None)
```

Comment:
Please update the documentation with this option.

Reply:
Added in [ce11791](https://github.com/matrix-org/synapse/pull/15644/commits/ce1179146fdeb7a66d0dd9afdad4da11e917e0d1)